src/z3c/checkversions/buildout.py

```diff
@@ -19,7 +19,12 @@
         "zc.buildout is not installed! \n"
         "If you're in a buildout environment,\n"
         "enable the buildout extra requirement like this:\n"
-        "eggs = z3c.checkversions [buildout]")
+        "eggs = z3c.checkversions [buildout]\n"
+        "If you installed z3c.checkversions via pip,\n"
+        "please first uninstall this package,\n"
+        "and then re-install it like this:\n"
+        "pip install z3c.checkversions[buildout]\n"
+        "or install zc.buildout separately.")
```

Comment:
There's no need to uninstall first.

Reply:
Ha, you are right - for pip this is not necessary, but it was for pipx, which I used.

At first I wanted to provide a message for both pip and pipx, but I removed the latter as I do not think that pipx is used broadly, and also we do not have to support all package managers.

Updated the message.